modules/oadp-features-plugins-known-issues.adoc

```diff
@@ -49,3 +49,22 @@ You can force the reconciliation by making any minor change to the DPA itself.
 ----
 $ oc -n openshift-adp get secret/oadp-<bsl_name>-<bsl_provider>-registry-secret -o json | jq -r '.data'
 ----
+
+
+[id="openshift-adp-controller-manager-seg-fault_{context}"]
+== `openshift-adp-controller` segmentation fault
+
+The `openshift-adp-controller-manager` pod fails with a crash loop segmentation fault, when the pod crashes and is restarted indefinitely until returning a segmentation fault, when configuring a DPA with `cloudstorage` and `restic` enabled.
+
+You can have either `velero` or `cloudstorage` defined, as they are mutually exclusive fields.
+
+* If you have both `velero` and  `cloudstorage` defined, the `openshift-adp-controller-manager` will fail.
+* If you have neither `velero` nor `cloudstorage` defined, the `openshift-adp-controller-manager` will fail.
+
+For more details about this issue, see link:https://issues.redhat.com/browse/OADP-1054[OADP-1054].
+
+
+[id="openshift-adp-controller-manager-seg-fault-workaround_{context}"]
+=== `openshift-adp-controller` segmentation fault workaround
```

Comment:
See comment above about heading guidelines.

Reply:
Does this really need a second heading? It seems like the standard SSG format would be sufficient: https://redhat-documentation.github.io/supplementary-style-guide/#_known_issue